simSPI/linear_simulator/projector.py

```diff
@@ -21,18 +21,144 @@ def __init__(self, config):
         super(Projector, self).__init__()
 
         self.config = config
-        self.vol = torch.rand([self.config.side_len] * 3, dtype=torch.float32)
-        lin_coords = torch.linspace(-1.0, 1.0, self.config.side_len)
-        [x, y, z] = torch.meshgrid(
-            [
-                lin_coords,
-            ]
-            * 3
-        )
-        coords = torch.stack([y, x, z], dim=-1)
-        self.register_buffer("vol_coords", coords.reshape(-1, 3))
+        self.space = config.space
+
+        if self.space == "real":
+            self.vol = torch.rand([self.config.side_len] * 3, dtype=torch.float32)
+            lin_coords = torch.linspace(-1.0, 1.0, self.config.side_len)
+            [x, y, z] = torch.meshgrid(
+                [
+                    lin_coords,
+                ]
+                * 3
+            )
+            coords = torch.stack([y, x, z], dim=-1)
+
+            self.register_buffer("vol_coords", coords.reshape(-1, 3))
+        elif self.space == "fourier":
+            # Assume DC coefficient is at self.vol[n//2+1,n//2+1]
```

Comment:
This comment could be moved in a docstring, so that it is more visible by users --- when we will publish the documentation website with Sphinx.

Reply:
This relates a bit to my comment in issue #102, we should try to standardize somewhere the conventions for Fourier space representations.  Then it doesn't really need to be anywhere...  I'm also not sure that this is currently accurate.